core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -375,1083 +380,674 @@ public boolean isPreloaded() {
             });
    }
 
-   @Override
-   public void disableStore(String storeType) {
-      if (enabled) {
-         boolean noMoreStores;
-         storesMutex.writeLock().lock();
-         publisherSemaphore.acquireUninterruptibly(Integer.MAX_VALUE);
+   private Single<?> preloadKey(long flags, MarshallableEntry me, DataConversion keyDataConversion, DataConversion valueDataConversion) {
```

Comment:
Is `preloadEntry` more accurate as we're executing `putKeyValueCommand`?

Reply:
Sure. This was just the prior name.